src/inline-help.scss

```diff
@@ -69,161 +68,7 @@ $block: #{$fd-namespace}-inline-help;
-    }
-
-    &--right {
-      top: -$fd-tooltip-padding;
-      left: $fd-tooltip-padding * 2.5;
-
-      &::before {
-        top: $fd-tooltip-arrow-offset + 8;
-        left: -($fd-tooltip-arrow-offset + 2);
-        transform: rotate(-90deg);
-      }
-
-      &::after {
-        top: $fd-tooltip-arrow-offset + 8;
-        left: -($fd-tooltip-arrow-offset + 1);
-        transform: rotate(-90deg);
-      }
-
-      @include fd-rtl() {
-        left: auto;
-        right: $fd-tooltip-padding * 2.5;
-
-        &::before {
-          left: auto;
-          right: -($fd-tooltip-arrow-offset + 2);
-          transform: rotate(-270deg);
-        }
-
-        &::after {
-          left: auto;
-          right: -($fd-tooltip-arrow-offset + 1);
-          transform: rotate(-270deg);
-        }
-      }
-    }
-
-    &--left {
-      top: -$fd-tooltip-padding;
-      right: $fd-tooltip-padding * 2.5;
-
-      &::before {
-        top: $fd-tooltip-arrow-offset + 8;
-        right: -($fd-tooltip-arrow-offset + 2);
-        transform: rotate(90deg);
-      }
-
-      &::after {
-        top: $fd-tooltip-arrow-offset + 8;
-        right: -($fd-tooltip-arrow-offset + 1);
-        transform: rotate(90deg);
-      }
-
-      @include fd-rtl() {
-        right: auto;
-        left: $fd-tooltip-padding * 2.5;
-
-        &::before {
-          right: auto;
-          left: -($fd-tooltip-arrow-offset + 2);
-          transform: rotate(270deg);
-        }
-
-        &::after {
-          right: auto;
-          left: -($fd-tooltip-arrow-offset + 1);
-          transform: rotate(270deg);
-        }
-      }
-    }
-
-    &--bottom-right {
-      left: -$fd-tooltip-arrow-offset;
-
-      @include fd-rtl() {
-        left: auto;
-        right: -$fd-tooltip-arrow-offset;
-      }
-
-      &::before {
-        top: -($fd-tooltip-arrow-offset);
-        left: $fd-tooltip-arrow-offset * 1.4;
-      }
-
-      &::after {
-        top: -($fd-tooltip-arrow-offset - 1);
-        left: $fd-tooltip-arrow-offset * 1.375;
-      }
-    }
-
-    &--bottom-left {
-      right: -$fd-tooltip-arrow-offset;
-
-      &::before {
-        top: -($fd-tooltip-arrow-offset);
-        right: $fd-tooltip-arrow-offset * 1.25;
-      }
-
-      &::after {
-        top: -($fd-tooltip-arrow-offset - 1);
-      }
-
-      @include fd-rtl() {
-        right: auto;
-        left: -$fd-tooltip-arrow-offset;
-
-        &::before {
-          right: auto;
-          left: $fd-tooltip-arrow-offset * 1.25;
-        }
-
-        &::after {
-          right: auto;
-          left: $fd-tooltip-arrow-offset * 1.25;
-        }
-      }
-    }
-
-    &--bottom-center {
-      right: -$fd-tooltip-arrow-offset * 20.25;
-
-      &::before {
-        top: -($fd-tooltip-arrow-offset);
-        right: $fd-tooltip-arrow-offset * 20.5;
-      }
-
-      &::after {
-        top: -($fd-tooltip-arrow-offset - 1);
-        right: $fd-tooltip-arrow-offset * 20.5;
-      }
-    }
```

Comment:
If you are migrating the component then you should get rid of the old variables `fd-` and use the theming parameters

Reply:
@athulvraj ping me if you need help with the theming parameters

Reply:
Sure, I will do